packages/joplin-desktop-app/joplin-desktop-app.pacscript

```diff
@@ -0,0 +1,31 @@
+name="joplin-desktop-app"
+gives="joplin-desktop"
+homepage='https://joplinapp.org/'
+pkgdesc="Joplin - the secure note taking and to-do app with synchronisation capabilities for Windows, macOS, Linux, Android and iOS"
+maintainer="lfromanini <lfromanini@yahoo.com>"
+repology=("project: joplin-desktop")
+arch=("amd64")
+pkgver="2.13.15"
```

Comment:
pkgver should be higher up.

Reply:
No, 2.13.15 is the stable one. The higher ones are pre-release.

Reply:
I meant the line should physically be higher up 😆